packages/components/addon/components/hds/app-footer/legal-links.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export default class HdsAppFooterLegalLinksComponent extends Component {
+  /**
+   * @param hrefForSupport
+   * @type {string}
+   * @description The href value of the "Support" link
+   */
+  get hrefForSupport() {
+    return this.args.hrefForSupport ?? 'https://www.hashicorp.com/support';
+  }
+
+  /**
+   * @param hrefForTerms
+   * @type {string}
+   * @description The href value of the "Terms" link
+   */
+  get hrefForTerms() {
+    return (
+      this.args.hrefForTerms ?? 'https://www.hashicorp.com/terms-of-service'
+    );
+  }
+
+  /**
+   * @param hrefForPrivacy
+   * @type {string}
+   * @description The href value of the "Privacy" link
+   */
+  get hrefForPrivacy() {
+    return this.args.hrefForPrivacy ?? 'https://www.hashicorp.com/privacy';
+  }
+
+  /**
+   * @param hrefForSecurity
+   * @type {string}
+   * @description The href value of the "Security" link
+   */
+  get hrefForSecurity() {
+    return this.args.hrefForSecurity ?? 'https://www.hashicorp.com/security';
+  }
+
+  /**
+   * @param hrefForAccessibility
+   * @type {string}
+   * @description The href value of the "Accessibility" link
+   */
+  get hrefForAccessibility() {
+    return (
+      this.args.hrefForAccessibility ?? 'mailto:accessibility@hashicorp.com'
```

Comment:
[Question] Just thinking out loud here, since the experience of this link is different from all others (open new page, vs open default client), would it be useful to add some sort of icon indicator here? Like an `external-link`? I don't have a strong opinion, just noting after clicking through all of the links.

Reply:
@andgen404 Let me know if you'd like to make any designs changes related to this.

Reply:
@heatherlarsen tagging you for visibility.

Reply:
We discussed this and agreed it isn't a block so I'm resolving. @andgen404